src/main/java/org/folio/innreach/domain/entity/InnReachTransaction.java

```diff
@@ -100,6 +106,11 @@
     "JOIN FETCH h.pickupLocation " +
     "WHERE h.folioLoanId = :folioLoanId AND irt.state NOT IN (4, 11, 12, 13)";
 
+  public static final String FETCH_BY_REQUEST_ID_QUERY_NAME = "InnReachTransaction.fetchTransactionItemHoldByHoldingsRecord";
+  public static final String FETCH_BY_REQUEST_ID_QUERY = "SELECT irt FROM InnReachTransaction AS irt " +
+    "JOIN FETCH irt.hold AS h " +
+    "JOIN FETCH h.pickupLocation " + "WHERE h.folioRequestId = :folioRequestId AND irt.state NOT IN (4, 11, 12, 13)";
```

Comment:
reuse FETCH_OPEN_TRANSACTIONS with appending condition for request id check

Reply:
Fixed

Reply:
Fixed